src/Sarif.WorkItems/SarifWorkItemFiler.cs

```diff
@@ -224,13 +237,18 @@ private void FileWorkItemsHelper(SarifLog sarifLog, SarifWorkItemContext filingC
                 //
                 UpdateLogWithWorkItemDetails(sarifLog, sarifWorkItemModel.HtmlUri, sarifWorkItemModel.Uri);
 
-                LogMetricsForFiledWorkItem(sarifLog, sarifWorkItemModel, FilingResult.Succeeded);
+                LogMetricsForProcessedModel(sarifLog, sarifWorkItemModel, FilingResult.Succeeded, null);
```

Comment:
>null [](start = 98, length = 4)

provide parameter name here for clarity (or make the underlying argument optional)